src/Format/MultipleDocumentationFormat.php

```diff
@@ -21,10 +23,19 @@ public function __construct(DocumentationFormat ...$documentationFormats)
         $this->documentationFormats = $documentationFormats;
     }
 
-    public function canHandler(SplFileInfo $file) : bool
+    /** @return string[] */
```

Comment:
Similar to above review: let's be as precise as we can with types: `@psalm-return list<string>` would be more close to what is being actually exposed